### PR TITLE
Doc: update the README guidelines to recent changes

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- [x] Update the readme with recent changes.
- [x] Update the layout of readme screenshots.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,20 +42,54 @@ You use a template to create a cheat sheet. Here is the procedure to follow:
...
+#. In ``make.bat``, add a new makefile command:
+
+   .. code:: bash
+
+         :<your_project>_cheat_sheet
+            pdflatex -output-directory=%BUILDDIR% cheat_sheats/<your_folder_name>/<your_tex_file_name>.tex --interaction=nonstopmode
+            goto end
+
+   Here, replace <your_project>, <your_folder_name>, and <your_tex_file_name> with your actual project name, folder name, and TeX file name, respectively.
+   This command uses latexmk to compile your TeX file into a PDF.
```
Comment (RobPasMue) [inline README.rst:74]:
```suggestion
   Here, replace ``<your_project>``, ``<your_folder_name>``, and ``<your_tex_file_name>`` with your actual project name, folder name, and TeX file name, respectively.
   This command uses latexmk to compile your TeX file into a PDF.
```

File: README.rst [hunk 2]
```diff
@@ -42,20 +42,54 @@ You use a template to create a cheat sheet. Here is the procedure to follow:
...
+   .. code:: bash
 
-        <your_project>_cheatsheat:
+        <your_project>_cheat_sheat:
 	        latexmk -f -pdf -use-make cheat_sheats/<your_folder_name>/<your_tex_file_name>.tex -cd -outdir=../../$(BUILD) -interaction=nonstopmode || true
            convert -density 150 -scene 1 $(BUILD_DIR)/<your_tex_file_name>.pdf $(BUILD_DIR)/<your_tex_file_name>.png
 
-Here, replace <your_project>, <your_folder_name>, and <your_tex_file_name> with your actual project name, folder name, and TeX file name, respectively.
-This command uses latexmk to compile your TeX file into a PDF, and then convert the first page of the PDF into a PNG file.
+   Here, replace <your_project>, <your_folder_name>, and <your_tex_file_name> with your actual project name, folder name, and TeX file name, respectively.
```
Comment (RobPasMue) [inline README.rst:58]:
```suggestion
   Here, replace ``<your_project>``, ``<your_folder_name>``, and ``<your_tex_file_name>`` with your actual project name, folder name, and TeX file name, respectively.
```

File: README.rst [hunk 3]
```diff
@@ -42,50 +42,98 @@ You use a template to create a cheat sheet. Here is the procedure to follow:
...
++=======================================+======================================+
+| |pyaedt API cheatsheet snapshot|      |   |pyedb API cheatsheet snapshot|    |
++---------------------------------------+--------------------------------------+
+
++--------------------------------------------+-----+
+| `Download PyPrimeMesh API cheatsheet pdf`_ |     |
++============================================+=====+
+| |PyPrimeMesh cheatsheet snapshot|          |     |
++--------------------------------------------+-----+
+
```
Comment (RobPasMue) [inline README.rst:116]: The rendering of this table is not great. Please work on it so that they have a similar distribution

**Review by RobPasMue — CHANGES_REQUESTED** [review]
Apart from my comment, I think that's all. I don't think it's necessary to include them in the main README... I would just put the links to the cheat sheats. You are already showing the preview in the actual docs site. Let's keep the README simple

**Comment by Revathyvenugopal162** [comment]
Thats true.I think i added it adapted from matplotlib, as there was a request to keep https://github.com/matplotlib/cheatsheets aligned with this cheatsheets. Also as an examples to others how to add this. I am open to change this honestly.

**Comment by RobPasMue** [comment]
> Thats true.I think i added it adapted from matplotlib, as there was a request to keep https://github.com/matplotlib/cheatsheets aligned with this cheatsheets. Also as an examples to others how to add this. I am open to change this honestly.

Yeah but they do not have a dedicated site where they put their cheat sheets AFAIK. So... I'd either remove them or properly format the tables. Up to you

**Comment by RobPasMue** [comment]
Add PyDPF Core cheat sheet whenever you can @Revathyvenugopal162 and let's clean up this PR according to our messages.

**Comment by Revathyvenugopal162** [comment]
In that case we can keep the readme simple.

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**